### PR TITLE
Address feedback from QA on homepage

### PR DESCRIPTION
**Pull request by tobyhede**
- [x] Ministers count should use use real count from DB
- [x]  Make news real
- [x] Lose the dummy consultation

### CODE REVIEW

File: app/views/templates/root_node.html.haml [hunk 1]
```diff
@@ -29,12 +29,14 @@
 
 
   %h2.home-heading News
-
+  - if NewsArticle.published.all.empty?
```
Comment (micapam) [inline app/views/templates/root_node.html.haml]: Feels wrong to have ActiveRecord stuff in a template. Maybe have `news_articles` getter in the RootNode class instead? Same for `ministers_count`

Comment (tobyhede) [inline app/views/templates/root_node.html.haml]: There was a conversation about that on Slack. I couldn't really decide where any of this should go.

Comment (micapam) [inline app/views/templates/root_node.html.haml]: RootNode may not be optimal but better than in the presentation layer IMO...

Comment (tobyhede) [inline app/views/templates/root_node.html.haml]: @micapam Pushed the methods to RootNode.

File: db/seeds.rb [hunk 2]
```diff
@@ -37,9 +37,8 @@
   section: topic,
   type: GeneralContent,
   state: :published
-}) do |node|
-  node.revise!(content_body: 'lorem ipsum').apply!
-end
+})
+node2.revise!(content_body: 'lorem ipsum').apply!
```
Comment (micapam) [inline db/seeds.rb]: Why this change?

**Comment by micapam** [comment]
Aside from my question on seeds.rb, this LGTM

File: db/seeds.rb (hunk 2 above)
Comment (micapam) [inline db/seeds.rb]: I think the error was the wrong node (`node` instead of `node2`) but it's a good idea to keep it in the block because if node2 already exists I guess we won't want to revise it? (Block on `find_or_create_by` only runs on create...)

**Comment by tessereth** [comment]
LGTM

**Comment by ltankey** [comment]
LGTM

File: db/seeds.rb (hunk 2 above)
Comment (tobyhede) [inline db/seeds.rb]: @micapam There was an error in the seeds. I think I understand what you are saying. Will address.
